CanvasCore/CanvasCore/Pages/UI/Rendering/PageTemplate.html

```diff
@@ -5,10 +5,10 @@
   <meta name="viewport" content="initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no" />
   <style>
     html {
-        background: {$BACKGROUND_COLOR$};
-        color: {$TEXT_COLOR$};
-        font-family: system-ui;
-        font-size: 16px;
+      background: {$BACKGROUND_COLOR$};
+      color: {$TEXT_COLOR$};
+      font-family: system-ui;
```

Comment:
I previously changed from `font: -apple-system-body` which would be a dynamic size to `font-family: system-ui` which supported more font weights. This template had an explicit `font-size: 16px`, which broke dynamic size.

I could have gone back to `-apple-system-body`, but it's base size is `17px` & that would make scaling everything much harder. We want a base `16px`, so we pass into these templates the size of our `regular16` scaled font.

Unlike UILabel, html won't resize automatically when the text size is updated regardless of which approach we take, so I settled on `system-ui` + the scaled font size, since it gives us the best font rendering we can get.